apps/fortran/multiDim/Makefile

```diff
@@ -12,7 +13,7 @@ OPS_GENERATED_SUFFIX=_ops
 APP = multidim
 MAIN_SRC=multidim
 
-TARGETS=seq tiled openmp mpi mpi_tiled mpi_openmp cuda mpi_cuda mpi_cuda_tiled
+TARGETS=seq tiled openmp mpi mpi_tiled mpi_openmp cuda mpi_cuda mpi_cuda_tiled ompoffload mpi_ompoffload mpi_ompoffload_tiled
```

Comment:
This is not necessarily required in Makefile. If not set in app Makefile, it will try to build all possible target as per compiler. This one declared in Makefile.c_app.
https://github.com/OP-DSL/OPS/blob/f7118855dc6bf9dcc3145ba3c0f72bb6655876d5/makefiles/Makefile.c_app#L74